bin/sstableloader

```diff
@@ -32,18 +35,18 @@ elif [ -r "$CASSANDRA_INCLUDE" ]; then
     . "$CASSANDRA_INCLUDE"
 fi
 
-if [ -z "$CLASSPATH" ]; then
```

Comment:
@tolbertam why have you removed all of this?

```
if [ -z "$CLASSPATH" ]; then
    echo "You must set the CLASSPATH var" >&2
    exit 1
fi

if [ "x$MAX_HEAP_SIZE" = "x" ]; then
    MAX_HEAP_SIZE="256M"
fi
```

Reply:
@tolbertam 

Aha I see, because it is all moved under tools/bin/sstableloader. Interesting. That way we might get rid of _everything_ above, no? Just check the script existence and then call it with all the args.

Reply:
Yeah, thats a good idea, keeps things simple to make it a dumb proxy, will change.

Reply:
ok just keep it like it is